.github/workflows/build-revive-wasm.yml

```diff
@@ -87,6 +87,7 @@ jobs:
           path: |
             ${{ env.REVIVE_WASM_INSTALL_DIR }}/resolc.js
             ${{ env.REVIVE_WASM_INSTALL_DIR }}/resolc.wasm
+            ${{ env.REVIVE_WASM_INSTALL_DIR }}/resolc_packed.js
```

Comment:
Does the suffix `packed` make sense here? Given that this is not really packed but loads the wasm file (i.e not containing it).

Reply:
Does not make sense. I am not sure if I will merge it, maybe I should find better solution

Reply:
Renamed to resolc_web.js as it is going to be used only from browser